Gruntfile.js

```diff
@@ -58,12 +58,18 @@ module.exports = function(grunt) {
 							slowTag = '--grep @slow';
 						} else if (tag === 'unstable') {
 							slowTag = '--grep @unstable';
+						} else if (tag === 'network') {
```

Comment:
In above block we have: 

```
if (tag === 'untagged') {
     slowTag = "--grep '@slow|@unstable' --invert";
}
```
which does to seems correct after introducing new tags. `Untagged` filter must be fixe and only run tests which are not tagged at all. If that's not the case then we should rename `Untagged` to something `default` or similar.

Reply:
Same as above